src/lib/build.cjs

```diff
@@ -34,6 +34,7 @@ const getBuildOptions = ({ cachedConfig, options: { context, cwd, debug, dry, js
   offline,
   cwd,
   featureFlags: {
+    edge_functions_config_export: true,
```

Comment:
I don't see the FF being added in the change. Is that on purpose?

Reply:
It was added in https://github.com/netlify/edge-bundler/pull/133.